components/tidb_query_codegen/src/rpn_function.rs

```diff
@@ -64,12 +64,16 @@
 //! validated. The validator function should have the signature `&tipb::Expr -> Result<()>`.
 //! E.g., `#[rpn_fn(raw_varg, extra_validator = json_object_validator)]`
 //!
-//! ### `metadata_ctor`
+//! ### `metadata_type`
+//! Type of raw metadata type defined in tipb.
```

Comment:
raw metadata is a confusing concept (what is raw?). Maybe call it `tipb metadata` is better, e.g.

The type of the metadata structure defined in tipb. If `metadata_mapper` is not specified, the protobuf metadata structure will be used as the metadata directly.